src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -230,6 +230,12 @@ public async Task<IEnumerable<string>> ExtractPackageAsync(string packagePath, D
             return allFilesInPackage;
         }
 
+        public async Task<IEnumerable<IPackageSearchMetadata>> GetLatestVersionsOfPackage(string packageId, bool includePreview, int numberOfResults)
```

Comment:
Doesn't a package only have one latest version? Or will this return both the latest stable and preview version?

Reply:
There's one latest version, but this is getting the latest, the second-latest, the third-latest, etc.

Reply:
It has an includePreview flag to determine whether it finds the absolute numberOfResults latest or the latest numberOfResults stable versions

Reply:
ah, for the `--take` parameter